client/src/index.tsx

```diff
@@ -32,8 +31,3 @@ ReactDOM.render(
   </React.StrictMode>,
   document.getElementById('root'),
 );
-
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
```

Comment:
The server worker was giving me errors in the new setup, and I don't think we were even using it, so it seemed worth removing.